recipes/pyslha/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pyslha" %}
+{% set version = "3.2.6" %}
+{% set python_min = "3.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pyslha-{{ version }}.tar.gz
+  sha256: 262a604c244db31a07362933001dedc24dd510e803785f9536e8d983ee66e75d
+  patches:
+    - use-setuptools-over-distutils.patch
+    - add-python_requires.patch
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - tex2pix >=0.1.5
+
+test:
+  imports:
+    - pyslha
+  commands:
+    - pip check
+
+    - test -f $PREFIX/bin/slha2isawig
+    - test -f $PREFIX/bin/isawig2slha
+    - test -f $PREFIX/bin/slhaplot
```

Comment:
With the tests below you don't need these, right?

Reply:
Yeah, that's fair. I generally do this just to verify at a glance of the recipe what the directory tree structure is going to be, but for this I don't really care so I'll remove these.